physics/lsm_ruc.meta

```diff
@@ -1344,13 +1398,37 @@
   kind = kind_phys
   intent = inout
 [snowfallac_lnd]
-  standard_name = surface_snow_amount_over_land
-  long_name = run-total snow accumulation on the ground over land
+  standard_name = surface_snow_amount_vardens_over_land
```

Comment:
I appreciate adding specificity to the metadata, but we try not to have arbitrary contractions (e.g. `varsdens`) in standard names. Could this (and also the similar ice-equivalent) be changed to `surface_snow_amount_assuming_variable_snow_density_over_land`?